### PR TITLE
Fix Portuguese translations

### PR DESCRIPTION
**Pull request by tuttiq**
Fixed Portuguese translations :)

Great work @theyokohamalife and @ann-kilzer !

### CODE REVIEW

File: locales/pt.json [hunk 1]
```diff
@@ -2,49 +2,45 @@
     "login": {
         "email": "E-mail",
         "password": "Senha",
-        "login": "Conecte-se",
-        "clear": "Claro"
-    },
-    "logout": {
```
Comment (ann-kilzer) [inline locales/pt.json:8]: Would you be able to merge with main? We might have had a merge conflict 😓

**Review by theyokohamalife — APPROVED** [review]
♥️Tutti ♥️ Thank you!!

**Review by ann-kilzer — CHANGES_REQUESTED** [review]
Thank you! 🙌🏻 

Could you merge with main to ensure you have the new "logout" object?

**Review by ann-kilzer — APPROVED** [review]
LGTM 🌟

**Pull request merged**